app/services/data_migrations/backfill_courses_for_next_cycle.rb

```diff
@@ -0,0 +1,10 @@
+module DataMigrations
+  class BackfillCoursesForNextCycle
+    TIMESTAMP = 20210917140806
+    MANUAL_RUN = false
+
+    def change
+      Course.where(recruitment_cycle_year: RecruitmentCycle.next_year).update_all(open_on_apply: true)
```

Comment:
```suggestion
      Course.where(recruitment_cycle_year: RecruitmentCycle.next_year, open_on_apply: false).update_all(open_on_apply: true, opened_on_apply_at: Time.zone.now)
```
Oops this completely slipped my mind, we need to set the `opened_on_apply_at` timestamp. Since we are setting it we also probably need to only set it for the closed on apply courses only.